config/database.py

```diff
@@ -0,0 +1,8 @@
+from pymongo.mongo_client import MongoClient
+
+uri = "mongodb+srv://admin:Password1@cluster0.4uf5yal.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0"
```

Comment:
We need to get credentials from a config file or sth. You may research available options about config management.